content/departments/engineering/dev/tools/grpc_cs_enablement.md

```diff
@@ -0,0 +1,110 @@
+# gRPC CS Enablement Guide
+
+# **What is gRPC?**
+
+[gRPC](https://grpc.io/) is a high-performance [RPC](https://en.wikipedia.org/wiki/Remote_procedure_call) framework developed by Google.
+
+## What is changing?
+
+In Sourcegraph `5.2.X`, we are enabling gRPC by default (with the option to disable it in case of an issue). gRPC will be the only option in the `5.3.X` release.
```

Comment:
Something in these two paragraphs should tell me that we use `gRPC` to communicate between internal services. That's an important bit of the context: we do NOT change external APIs, we only change the way our services communicate with each other.